css-fonts/styles.css

```diff
@@ -0,0 +1,4 @@
+body {
+  font-family: 'Fira Sans', sans-serif;
```

Comment:
You should be using 2 separate fonts. One for the headings, and one for the rest of the content